src/main/python/vantiqsdk.py

```diff
@@ -1478,7 +1486,8 @@ async def subscribe(self, path: str, params: dict, callback: Callable[[str, dict
         if target_namespace is not None:
             request_id += '@' + target_namespace
 
-        if request_id in self.subscriptions.keys():
+        # Check to see if we already have a subscription for this path
```

Comment:
There was a failure mode here where we would attempt the subscription, it would fail with an exception and then we're stuck in a state where we can't re-issue it since the `False` entry blocks that.